docs/data-sources/eps.md

```diff
@@ -0,0 +1,51 @@
+---
+subcategory: "Enterprise Project Management Service (EPS)"
+---
+
+# huaweicloud\_enterprise\_project
+
+Use this data source to get an enterprise project from HuaweiCloud
+
+## Example Usage
+
+```hcl
+data "huaweicloud_enterprise_project" "test" {
+  name = "test"
+}
+```
+
+## Resources Supported Currently:
+Service Name | Resource Name
+---- | ---
+VPC | huaweicloud_vpc<br>huaweicloud_vpc_eip<br>huaweicloud_vpc_bandwidth<br>huaweicloud_networking_secgroup
+ECS | huaweicloud_ecs_instance<br>huaweicloud_compute_instance
```

Comment:
huaweicloud_ecs_instance is deprecated, just ignore it here.

Reply:
fixed